docs/reference/function-configuration/stream-response.md

```diff
@@ -0,0 +1,115 @@
+# Streaming HTTP Responses
```

Comment:
I think the `tasks` dir suits better for the location of this doc, since it doesn't really touch function configuration at all.

Reply:
@TomerShor it depends what to consider a "function configuration". As a user, I'd assume that this is generally about my function including the user code probably. We have batching and async already in this dir